core/src/main/java/org/pentaho/di/core/database/Database.java

```diff
@@ -597,6 +597,8 @@ private void connectUsingClass( String classname, String partitionId ) throws Ke
 
         connection = DriverManager.getConnection( url, properties );
       }
+    } catch ( SQLException sqlException ) {
+      throw new KettleDatabaseException( BaseMessages.getString( PKG, "Database.Exception.ConnectionTestFailed", toString() ), sqlException );
```

Comment:
Since the Database class is generally used (not just with connection tests), I'm thinking the message should probably not refer to "tests".  Maybe just "Connection failed. Verify all connection parameters, and confirm the appropriate driver is installed.".  We could also include the URL in the message, though we'd definitely want to mask the password.

Would also be good to run whatever message by Katelyn V. to make sure Ux likes the word choice.

Reply:
Agreed, I just pushed the updated text after touching base with Katelyn V.